example-data/turtle/geneticDiagnosis.ttl

```diff
@@ -53,5 +53,5 @@
     sio:SIO_000300 "https://identifiers.org/HGNC:12698"^^xsd:string ;
     sio:SIO_000628 <https://identifiers.org/HGNC:12698> .
 
-<https://identifiers.org/HGNC:12698> a sio:SIO_000614, sio:SIO_000015, obo:NCIT_C173071 ;
+<https://identifiers.org/HGNC:12698> a sio:SIO_000614, sio:SIO_000015, obo:NCIT_C171178 ;
```

Comment:
This node is of type gene right? Why `obo:NCIT_C171178 (Sequence Variant Report)` used as a type here?

Reply:
Based on current YARRRML templates, all three (HGVS, OMIM and HGNC) have `obo:NCIT_C171178 (Sequence Variant Report)` as type